src/components/scene/Game/Game.tsx

```diff
@@ -0,0 +1,31 @@
+import { Suspense } from 'react';
+import { Canvas } from '@react-three/fiber';
+import * as constants from '../constants';
+import './Game.css';
+import Board from '../Board';
+import Lava from '../Lava';
+
+const Game = () => {
+  return (
+    <div id="game">
+      <Canvas
+        flat
+        linear
+        camera={{
+          position: constants.CAMERA_DEFAULT_POSITION
+        }}
+      >
+        <Suspense>

```

Comment:
why do we need <Suspense> here if there is no fallback specified?

Reply:
💀